klippy/extras/temperature_combined.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+# Support for combination of temperature sensors
+#
+# Copyright (C) 2023  Michael Jäger <michael@mjaeger.eu>
+#
+# This file may be distributed under the terms of the GNU GPLv3 license.
+
+UPDATE_TEMPERATURE_INTERVALL = 1.0
```

Comment:
FYI, there's a typo here, which may prevent merging:  It should be spelled `UPDATE_TEMPERATURE_INTERVAL`.

Reply:
Fixed via 5013943